flycheck.el

```diff
@@ -934,6 +934,25 @@ currently listed."
   :risky t
   :package-version '(flycheck . "0.20"))
 
+(defcustom flycheck-automatically-display-error-at-point t
+  "Automatically display display error at point when available.
+
+If t, use `flycheck-display-error-at-point-timer' to show error under point
+after commands, and `flycheck-idle-change-timer' after changes.
+
+Otherwise disable all unnecessary timers to suppress messages."
+  :group 'flycheck
+  :type 'boolean
+  :risky t
+  :package-version '(flycheck . "0.31")
+  :set
```

Comment:
Please remove the whole `:set`, we will just use the value when activating/deactivating the minor mode instead.